src/main/java/frc/robot/subsystems/SwerveDrivetrain.java

```diff
@@ -61,10 +61,9 @@
         private double currentHeading; // for simulated current gyro reading (yaw)
         private AHRS m_gyro;
         private SwerveDriveOdometry m_odometry;
-        private SwerveDriveKinematics m_kinematics;
+        public SwerveDriveKinematics m_kinematics;
```

Comment:
why?

Reply:
is so we can use m_kinematics in robot container without having to make a getKinematics() method

Reply:
but why would a `getKinematics()` method be bad?

Reply:
cuz typing = bad

i know we're a great programming team

Reply:
then dictate if you dont want to type?

Reply:
Should be fixed